typing/printtyp.ml

```diff
@@ -277,10 +277,21 @@ let rec normalize_type_path ?(cache=false) env p =
   with
     Not_found -> (p, Id)
 
+let penality s =
```

Comment:
I think this should be called `penalty`. The same typo exists in the `Changes` file.